Hunting Queries/SAP/SAP_HighdownloadfromPriviledgedaccount.yaml

```diff
@@ -0,0 +1,52 @@
+id: 2843e796-3d6c-4a78-a815-1db783b346a3
+name: High count download from a SAP Privileged account
+description: |
+  'This query detects high counts of download from a sensitive SAP Privileged account. A pre-built watchlist is leveraged to identify the privileged users that are under extra restrictions.
+  Reference: 
+    https://learn.microsoft.com/en-us/azure/sentinel/sap/sap-solution-security-content#available-watchlists
+    https://techcommunity.microsoft.com/t5/azure-sentinel/what-s-new-watchlists-templates-are-now-in-public-preview/ba-p/2614340'
+requiredDataConnectors:
+  - connectorId: SAP
+    dataTypes:
+      - SAPAuditLog
+tactics:
+  - InitialAccess
+  - Exfiltration
+relevantTechniques:
+  - T1078
+  - T1030
+query: |
+    let priv_users = ('_GetWatchlist("VIPUsers")') ;
```

Comment:
priv_users should be list of distinct users. Fetch results from VIPUsers watchlist and perform distinct operation on User Principal Name.
Schema: https://learn.microsoft.com/en-us/azure/sentinel/watchlist-schemas